src/components/CallView/shared/VideoVue.vue

```diff
@@ -79,6 +79,9 @@
 		</slot>
 		<div v-if="isSpeaking && !isStripe && !isBig" class="speaking-shadow" />
 		<div v-if="!unSelectable && mouseover && !isBig" class="hover-shadow" />
+		<div v-if="isPresenterOverlay && mouseover" class="hover-presenter">
+			<HideIcon fill-color="#ffffff" @click="handleClickVideo" />
```

Comment:
```suggestion
			<HideIcon fill-color="#ffffff" />
```
Doesn't work for you like this?

Reply:
IMHO it's better to replace all these 3 divs with simple classes (and maybe `::after` pseudo-elements), as they're adding only shadow or background. Can also use `background-image: var(--icon-toggle-white);` for the eye icon, although it's not crossed, as MDI component:
![image](https://github.com/nextcloud/spreed/assets/93392545/fbdc45b4-e3c9-41ab-a907-51f6d3d9af0c)